core/src/net/sf/openrocket/file/openrocket/savers/RocketComponentSaver.java

```diff
@@ -90,9 +90,8 @@ protected void addParams(net.sf.openrocket.rocketcomponent.RocketComponent c, Li
 			
 			if( c instanceof Clusterable ){
```

Comment:
Is this still needed?

Reply:
yup.  There's a comment the line after that which explains why.  

Also, that version was buggy. Try the new version :)

Reply:
Ah, got it.  It was the bugginess that threw me -- emitInteger wasn't in the else clause with that version.

Reply:
Yea.  That was definitely a bug.